tests/integration_tests/validation/validation_util.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pytest
+
+from arelle.CntlrCmdLine import parseAndRun
+from arelle import ModelDocument
+
+
+def get_test_data(args):
+    """
+    Produces a list of Pytest Params that can be fed into a parameterized pytest function
+
+    :param args: The args to be parsed by arelle in order to correctly produce the desired result set
```

Comment:
Not sure if this is conventional within the larger python community. I did some looking and I am not sure they have a standard so I would be inclined to do whatever the ide auto generates